webpack.config.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
       },
       {
         test: /\.jsx?$/,
-        loader: 'babel-loader',
+        loader: 'babel-loader?presets[]=react',
```

Comment:
I guess my thought was just to avoid bringing in unnecessary Babel transforms in general. We can define the preset in .babelrc if you'd rather not clutter up webpack's config. What do you think?